platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuildingBlocks.java

```diff
@@ -63,6 +64,6 @@ public record PlatformBuildingBlocks(
         @NonNull ReservedSignedState initialState,
         @NonNull EmergencyRecoveryManager emergencyRecoveryManager,
         @NonNull Consumer<GossipEvent> preconsensusEventConsumer,
-        @NonNull Consumer<ConsensusSnapshot> snapshotOverrideConsumer,
-        @NonNull IntakeEventCounter intakeEventCounter,
+        @Nullable Consumer<ConsensusSnapshot> snapshotOverrideConsumer,
+        @Nullable IntakeEventCounter intakeEventCounter,
```

Comment:
You accidentally marked `intakeEventCounter` `@Nullable`, instead of `preconsensusEventConsumer`

Reply:
oops, fixed